h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -59,7 +59,8 @@ class SearchBarController extends Controller {
         {
           matchOn: 'url',
           title: 'url:',
-          explanation: 'see all annotations on a page',
+          explanation: `see all annotations on page(s) * matches any number 
+            of characters and ? matches any single character`,
```

Comment:
This is missing a period at the end of the first sentence (after "page(s)"), which makes it a little difficult to parse:

<img width="672" alt="screenshot 2018-09-21 13 26 04" src="https://user-images.githubusercontent.com/2458/45881076-ff92df80-bda1-11e8-8cda-b4adde6aaaeb.png">

Perhaps it might also be more consistent to change the first sentence to "search by document URL". I'm assuming here that the majority of visitors will know what a "URL" is.